synapse/rest/__init__.py

```diff
@@ -95,42 +99,53 @@ def register_servlets(client_resource: HttpServer, hs: "HomeServer") -> None:
         v1_login.register_servlets(hs, client_resource)
         profile.register_servlets(hs, client_resource)
         presence.register_servlets(hs, client_resource)
```

Comment:
I wonder how are all these servlets are ordered?

Reply:
(Probably just the order they were written in)

Reply:
I think they're arbitrary. I could alphabetize them if you'd like?

Reply:
(I have some follow-ups anyway though so will probably hold off for now.)

Reply:
No strong opinions, just thinking (typing) aloud again. Let's ship it!